File: Easy-Array/Union.java
import java.util.ArrayList;
import java.util.HashSet;
import java.util.Scanner;

class Union 
{
    public static ArrayList<Integer> findUnion(int[] a, int[] b) 
    {
       ArrayList<Integer> temp = new ArrayList<Integer>();
       for(int i=0;i<a.length;i++)
       {
         temp.add(a[i]);
       }
       for(int i=0;i<b.length;i++)
       {
         temp.add(b[i]);
       }
       HashSet<Integer> cars = new HashSet<Integer>();
       for(int i=0;i<temp.size();i++)
       {
         cars.add(temp.get(i));
       }
       ArrayList<Integer> result = new ArrayList<Integer>(cars);
       return result;
    }
    public static void main(String[] args)
    {
       Scanner sc=new Scanner(System.in);
       System.out.println("Size of Array one");
       int n=sc.nextInt();
       int a[]=new int[n];
       System.out.println("Enter the elements of array");
       for(int i=0;i<n;i++)
       {
        a[i]=sc.nextInt();
       }
       System.out.println("Size of Second Array");
       int m=sc.nextInt();
       int b[]=new int[m];
       System.out.println("Enter the elements of array");
       for(int i=0;i<m;i++)
       {
         b[i]=sc.nextInt();
       }
       ArrayList<Integer> result = findUnion(a,b);
       System.out.println("Union of two array is "+result);

    }
}

File: Easy-Array/Union.py
# WAP to find the union of two arrays. The union of two arrays is a list that contains all the unique elements from both arrays. For example, if the first array is [1, 2, 3] and the second array is [3, 4, 5], then the union of these two arrays would be [1, 2, 3, 4, 5].


def union_of_arrays(arr1, arr2): 
    # Create a set to store unique elements
    union_set = set() 

    # Add elements of the first array to the set
    for num in arr1: 
        union_set.add(num) 

    # Add elements of the second array to the set
    for num in arr2: 
        union_set.add(num) 

    # Convert the set back to a list and return it
    return list(union_set)

num=int(input("Enter the number of elements in the first array: "))
arr1 = []
print("Enter the elements of the first array: ")
for i in range(num):
    arr1.append(int(input()))
num=int(input("Enter the number of elements in the second array: "))
arr2 = []
print("Enter the elements of the second array: ")
for i in range(num):
    arr2.append(int(input()))

result = union_of_arrays(arr1, arr2)
print("The union of the two arrays is: ", result)